include/envoy/upstream/load_balancer_type.h

```diff
@@ -29,6 +29,7 @@ struct SubsetSelector {
   std::set<std::string> selector_keys_;
   envoy::api::v2::Cluster::LbSubsetConfig::LbSubsetSelector::LbSubsetSelectorFallbackPolicy
       fallback_policy_;
+  std::set<std::string> fallback_keys_subset_;
```

Comment:
Per the main review comment, I think it's time SubsetSelector was promoted to an abstract class with separate implementation.

Reply:
done